README.md

```diff
@@ -4,6 +4,23 @@ Dummy credentials repository used to generate organisation specific credential r
 
 **NOTE:** This repository serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) repository.
 
+## Reporting changes back
```

Comment:
Would you mind moving this to the end of the README please? I think it makes sense to leave Getting Started as the first section here

Reply:
I see where you are coming from but I disagree. You want people to be reminded they have to report changes back. I feel that if this is moved to the bottom some people may read and not remember to push the changes back. If its at the start it puts the idea of reporting changes back in their mind even if they only read the title of the section.

I will gladly change this if you wise, I just wanted you to understand my point of view.

To be clear, Do you want me to move this to the bottom?

Reply:
@Boomatang Fully understood and I agree with your points here. Perhaps as a compromise we could have a **Prerequisites** heading at the top of the README and move **Reporting changes back** as a subheading under this section? That way the flow would be as follows (which to me makes more sense):

- Prerequisites
-- Reporting changes back
- Getting Started
- .........

Reply:
Yap. That makes sense to me. I will move it under Prerequisites.